### ISSUE TITLE
Core association is forced in Playlists

### ISSUE THREAD

**Issue by andiandi13** [libretro/RetroArch#12243]
## Description

I created playlists with no Default Core (N/A in playlist manager). Also all my .lpl files have "DETECT" written on both core path and core name lines.

But if I load a game, say Golden Eye in my Nintendo 64 playlist with _ParaLLEIL N64_, it will be the default core unless I Reset the core association for each game _or_ for each playlist in the playlist manager.

![RAcore](https://user-images.githubusercontent.com/19387072/113770552-8a4e4680-9722-11eb-9e3e-187e32cf3bfa.gif)

I looked at RA options for hours and search and forums and issues on here but didn't find anything about that.

### Expected behavior

I want to be asked to choose a core **each and every time I run a game** on my playlists, just like with "Load Content'.

What would be even better, is if on the top list of suggested cores for each game, is the last used core of that playlist so that if we still want to use the same core, it'd be quick while having the choice at the same time.

### Actual behavior

The last used core is set as default per game in a playlist

- RetroArch: [1.9.1]

### Environment information

- OS: [Win 7 x64]

**Comment by hizzlekizzle** [comment]
This is working as intended. If you want it to stay DETECT forever, you have to make the playlist file read-only.

**Comment by andiandi13** [comment]
@hizzlekizzle Making playlists read-only is far from being convenient when you edit your playlists quite often + you have to remember that you've done that when your playlists won't update.

Having a forced core can be annoying. In my case for example, I take N64 screenshots with a core set on Angry Lion (software emulation), and I switch to a core set on hardware settings to be able to play at full speed. But I have to Reset core association everytime.
We can imagine more situations where it can be annoying like if you want to compare which core runs better for each game (e.g. NDS), you'll have to reset association 20 times if you switch cores that much to test.

I think it'd be easy to add an option either in Settings > Playlists to disable that behavior
Or in each Playlist settings in the Playlists manager, in _Default Core_.
Currently there's N/A that tranform _DETECT_ to a specific core each time you run a game

**_Why not renaming that entry _Auto_ and making a new _N/A_ entry that will keep DETECT lines on .lpl files whatever happens ?_**

**Comment by hizzlekizzle** [comment]
Or you could just use 'load content' for testing and save playlists for casually playing (that's what I do, whenever I use playlists at all)

**Comment by andiandi13** [comment]
Yes Load content can be a workaround. But if you have you roms split inside two hard drives it's long to navigate (which is another issue, not being able to set any folder as a shortcut).
